docs/proposals/scheduling/aggregate-preference/aggregate-preference.md

```diff
@@ -0,0 +1,256 @@
+        - member1
+        - member2
+    replicaScheduling:
+      replicaSchedulingType: Divided
+      replicaDivisionPreference: Aggregated
+      aggregatePreference:
+        preferredAggregationList:
+          - targetCluster:
+              clusterNames:
+                - member1
+            priority: 2
+          - targetCluster:
+              clusterNames:
+                - member2
+            priority: 1
+```
+
+As you see, the user specifies the cluster aggregation priority through `spec.placement.replicaScheduling.aggregatePreference` 
+field, the member1 cluster has a higher priority and will be aggregated first.
+
+#### Controller logic changes
+
+The logic of the scheduler refers to the static weight scheduling. Before `selectClusters` step, it will still 
+go through `filter` and `score` plug-in:
+
+* The `filter` plug-in will take effect, and the feasible clusters will be the filtered clusters basing on user provided clusters.
+* The `score` plug-in will not take effect, and clusters will be sorted according to the priority specified by the user.
+
+As for `assignReplicas` step, the detail assign methods remain unchanged, but the order of the input available clusters is different.
+
+* Previous: first sort by `MaxAvailableReplicas`, and then resort to prioritize the scheduled clusters in advance.
+* Now: by user specified.
+
+#### Corner case
+
+Supposing the user has two clusters: member1 (priority=2) and member2 (priority=1)
+
+In this scheduling type, a deployment is assigned to member2 when first propagation, for the reason of member1 is filtered
+due to some reason. Then, this deployment scaling up replicas after member1 cluster recovered, at this time, 
+should the new replicas be aggregated to previous member2 cluster or be assigned to member1 cluster?
+
+If it is the former, it violates the user's priority, while if it is the latter, it violates the concept of aggregation.
+
+### Solution two: Using multiple scheduling group
+
+#### API changes
+
+The current design of multiple scheduling groups cannot meet this requirement, mainly due to when choosing next clusterAffinity,
+we treat all clusters in that group equally. However, in this requirement, it is expected to prioritize the clusters which
+already appeared in the previous clusterAffinity.
+
+Considering compatibility with previous multiple scheduling group, we introduce a new field `clusterAffinitiesOrder` to
+`placement` of policy and binding. When it equals to value `sequential`, we would prioritize the clusters which already 
+appeared in the previous clusterAffinity. When left empty, we treat all clusters in that group equally.
+
+```go
+type Placement struct {
+    // ...
+    // +optional
+    ClusterAffinities []ClusterAffinityTerm `json:"clusterAffinities,omitempty"`
+
++   // ...
++   // +kubebuilder:validation:Enum=sequential
++   // +optional
++   ClusterAffinitiesOrder ClusterAffinitiesOrder `json:"clusterAffinitiesOrder,omitempty"`
```

Comment:
Could explicit priority declaration for clusters be implemented, considering the following points:
- We can define the priority of clusters as an inherent attribute of the clusters themselves. Various scheduling methods would then act merely as consumers of this data, avoiding binding them directly to any specific scheduling approach.
- The current Solution 2 has coupling between different clusterAffinities, which requires determining the priority of clusters based on the order in which clusters appear within the clusterAffinity. Additionally, it does not allow for distinguishing the priority relationship between clusters that appear for the first time within the same clusterAffinity.